src/handlers/motions/motionCreated/handlers/metadataDelta.ts

```diff
@@ -47,6 +48,15 @@ export const handleMetadataDeltaMotion = async (
         gasEstimate: gasEstimate.toString(),
       });
     }
+
+    if (isRemoveVerifiedMembersOperation(operation)) {
+      await createMotionInDB(event, {
+        type: ColonyActionType.RemoveVerifiedMembersMotion,
+        members: operation.payload,
+        pendingColonyMetadataId,
```

Comment:
Do you know if this is needed @iamsamgibbs? Not sure if I'm missing something but none of the CDapp sagas seem to be creating the metadata it would link to.

Reply:
I don't believe it is needed actually, no references to it in the CDapp sagas so I've just removed it from both add and remove and still working fine.